prism-languages.php

```diff
@@ -45,7 +45,6 @@ function mkaz_code_syntax_block_get_supported_languages() {
 		"toml" => "TOML",
 		"typescript" => "TypeScript",
 		"vim" => "vim",
```

Comment:
:warning: **Warning**: Array double arrow not aligned correctly; expected 8 space(s) between &quot;&quot;vim&quot;&quot; and double arrow, but found 1 (*WordPress.Arrays.MultipleStatementAlignment.DoubleArrowNotAligned*).